api_app/choices.py

```diff
@@ -32,11 +32,24 @@ def get_priority(cls, tlp):
 class Status(models.TextChoices):
     PENDING = "pending", "pending"
     RUNNING = "running", "running"
+
+    ANALYZERS_COMPLETE = "analyzers_complete", "analyzers_complete"
```

Comment:
is maybe better to use `..ers_completeD`? Same for all the others.

Plus, would make sense to add also "CONNECTORS_RUNNING" and "VISUALIZERS_RUNNING" or do you think that this is implicit in the previous statuses?

Reply:
It is kinda implicit, but I can see users that does not know the framework well enough to ask themselves "what is going on?"
For this reason i think that adding the intermediary status, its actually a good idea, and does not provoke any side effect at all.